src/main/java/com/google/api/codegen/transformer/java/JavaSchemaTypeNameConverter.java

```diff
@@ -146,7 +146,7 @@ public TypeName getTypeNameForElementType(TypeModel type) {
   private TypeName getTypeNameForElementType(
       DiscoveryField fieldModel, BoxingBehavior boxingBehavior) {
     if (fieldModel == null) {
-      return new TypeName("java.lang.Void", "Void");
+      return new TypeName("com.google.api.gax.httpjson.EmptyMessage", "Void");
```

Comment:
Should the second argument be replaced with `"EmptyMessage"` (now it is still `"Void"`)?

Reply:
Oh yes. Thanks.